packages/react-dev-utils/paths/removeFileNameHash.js

```diff
@@ -0,0 +1,7 @@
+// Input: /User/dan/app/build/static/js/main.82be8.js
+// Output: /static/js/main.js
+module.exports = paths => function removeFileNameHash(fileName) {
```

Comment:
There is no need for this to be a separate file. Please move it to `printFileSizes`.